### DISCUSSION TITLE
Addition of Quick Start How to Build

### DISCUSSION THREAD

**Discussion by aaramians-hsl** [MarkMpn/Sql4Cds#276] [General]
Any chance for a quick start guide on what are the steps to build and debug?
looking for a way to build and integrate/run this with XrmToolBox to potentially add some extension, I am interested in adding a support for connection to local MS-SQL database as additional data source.

**Comment by MarkMpn** [comment]
You should be able to just open the solution in Visual Studio and build it - are you hitting some sort of error?
